### DISCUSSION TITLE
DSEP Working Group Kick-off Call

### DISCUSSION THREAD

**Discussion by ravi-prakash-v** [beckn/DSEP-Specification#13] [Announcements]
**Greetings Folks!**

Introducing the **Decentralized Skilling and Education Protocol (DSEP)**.

It is an open-source community-owned project provides open interoperable specifications for creating decentralized skills and education networks. It is an adaptation of beckn protocol core specification with added taxonomies and sample network policies for the **Skills and Education** sector.

Adoption of the protocol can only be as good as the ease of implementation of the protocol. And if this adoption must cater to a global community, that challenge is a lofty one. Such a challenge requires an in-depth understanding of beckn protocol and a zero-to-one approach to solving technical problems from first principles.

Today we are kick-starting the **DSEP Working Group** at **4:00 pm IST**.  This group maintains the DSEP specification repository and continuously strives to evolve the specification by accepting contributions from the community.

This is an invitation to all the energetic people in the community to join the DSEP Working Group as volunteers, and collectively work together to enable the Education, Skilling and Employment ecosystem to adopt the specification; innovate and build rich, integrated products.

To join the working group, please fill up this form and we will get in touch with you soon!

https://docs.google.com/forms/d/e/1FAIpQLSexE0GdpRoSSF0DA-SDz0wl4NHrfXHXJ1mzKkSXQYyhhCbe8g/viewform?usp=sf_link

**Comment by sujitrs** [comment]
Please find meeting artefacts @ 09 Feb 2022. Attached artefact consists of Presentation, Attendees, Action Items and Meeting Notes. Meeting Recording has failed.
[Meeting Artefacts @ 9 Feb 2022.pdf](https://github.com/beckn/DSEP-Specification/files/8078320/Meeting.Artefacts.%40.9.Feb.2022.pdf)
